moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -262,11 +262,11 @@ const ProjectComponentEdit = ({
     const selectedType = (newValue ?? e.target.value ?? "").toLowerCase();
 
     // Generates a list of available component subtypes given a component type
-    const availableSubTypes = getAvailableSubtypes(selectedType);
+    const newAvailableSubTypes = getAvailableSubtypes(selectedType);
```

Comment:
ah, was it conflicting with state?

Reply:
Thankfully not, but it made it difficult to troubleshoot because the names were nearly identical.